src/main/java/com/exasol/RequestDispatcher.java

```diff
@@ -0,0 +1,91 @@
+package com.exasol;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import org.apache.commons.cli.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.exasol.github.*;
+import com.exasol.release.ReleasePlatform;
+
+/**
+ * This class is the main entry point for calls to a Release Robot.
+ */
+public class RequestDispatcher {
+    private static final Logger LOGGER = LoggerFactory.getLogger(RequestDispatcher.class);
+    private static final String REPOSITORY_OWNER = "exasol";
+
+    /**
+     * Main entry point for all Release Robot's calls.
+     * 
+     * @param repositoryName name of a target project from GitHub
+     * @param goalAsString run goal. Supported goals: release, validate
+     * @param platforms one or more platforms for validation or release. Supported values: github
+     */
+    public void dispatch(final String repositoryName, final String goalAsString, final String... platforms) {
+        LOGGER.info("Release Robot has received '{}' request for the project '{}'.", goalAsString, repositoryName);
+        try {
+            final Goal goal = getGoal(goalAsString);
+            final Set<ReleasePlatform> platformsList = getReleasePlatformsList(platforms);
+            final GitHubRepository repository = GitHubRepositoryFactory.getInstance()
+                    .createGitHubRepository(REPOSITORY_OWNER, repositoryName);
+            final RepositoryHandler repositoryHandler = new RepositoryHandler(repository, platformsList);
+            if (goal == Goal.VALIDATE) {
+                repositoryHandler.validate();
+            } else {
+                repositoryHandler.validate();
+                repositoryHandler.release();
+            }
+        } catch (final RuntimeException exception) {
+            LOGGER.error("'{}' request failed. Cause: {}", goalAsString, exception.getMessage());
+        }
+    }
+
+    protected Goal getGoal(final String goalAsString) {
+        try {
+            return Goal.valueOf(goalAsString.toUpperCase().trim());
+        } catch (final IllegalArgumentException illegalArgumentException) {
+            final List<String> allowedGoals = Arrays.stream(Goal.values()).map(goal -> goal.toString().toLowerCase())
+                    .collect(Collectors.toList());
+            LOGGER.error("Cannot parse a goal '{}'. Please, use one of the following goals: {}", goalAsString,
+                    String.join(",", allowedGoals));
+            throw illegalArgumentException;
+        }
+    }
+
+    protected Set<ReleasePlatform> getReleasePlatformsList(final String[] platforms) {
+        final Set<ReleasePlatform> platformsList = new HashSet<>();
+        for (final String platform : platforms) {
+            platformsList.add(ReleasePlatform.valueOf(platform.toUpperCase()));
```

Comment:
I'm afraid `valueOf` is case-sensitive and we should keep this case transformation for user's convenience. 

`It's important to remember that the valueOf() method does a case-sensitive match of the argument supplied to it, so passing a value that doesn't match the case of any of the original enum‘s values would lead to an IllegalArgumentException:`

Quote from: https://www.baeldung.com/java-string-to-enum